### PR TITLE
Move event alias mappings to their components

### PR DESCRIPTION
**Pull request by derrabus**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | N/A
| License       | MIT
| Doc PR        | N/A

For a small console tool, I wanted to use the Console, EventDispatcher and DI components standalone. In order to make use of aliased events, I had to copy over the mapping information from FrameworkBundle, which was a bit unfortunate.

With this PR, I'd like to suggest to store all alias mappings on the `*Events` classes of each component instead. This change will make the mapping reusable outside of the full-stack framework.

Note: I've bumped the dependencies of FrameworkBundle/SercurityBundle in order to gain access to the moved mapping information. If any of the bumps is too heavy, please tell me and I'll implement a fallback instead.

### CODE REVIEW

**Review by wouterj — APPROVED** [review]
I like this idea.

I think the version constraints are fine. I believe bundle constraints can be quite strict, it's weird to use a 5.2 bundle in a <5.2 Symfony application.

File: src/Symfony/Component/Console/Tests/ConsoleEventsTest.php [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+use Symfony\Component\Console\Tester\ApplicationTester;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Reference;
+use Symfony\Component\EventDispatcher\DependencyInjection\RegisterListenersPass;
+use Symfony\Component\EventDispatcher\EventDispatcher;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+class ConsoleEventsTest extends TestCase
+{
+    public function testEventAliases(): void
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Console/Tests/ConsoleEventsTest.php:32]: ~`: void`~

**Review by nicolas-grekas — APPROVED** [review]

File: src/Symfony/Component/Console/Tests/ConsoleEventsTest.php (hunk 1 above)
Comment (derrabus) [reply src/Symfony/Component/Console/Tests/ConsoleEventsTest.php]: Done.

**Comment by derrabus** [comment]
The failure on Travis looks unrelated.

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @derrabus.

**Pull request merged**